packages/common/src/styles/theme.ts

```diff
@@ -41,10 +44,25 @@ declare module '@mui/material/styles/createMixins' {
   }
 }
 
+declare module '@mui/material/Checkbox' {
+  export interface CheckboxPropsColorOverrides {
+    darkGrey: true;
+  }
+}
+
+declare module '@mui/material/styles' {
+  interface Palette {
+    darkGrey: Palette['primary'];
+  }
+  interface PaletteOptions {
+    darkGrey: PaletteOptions['primary'];
+  }
+}
```

Comment:
Because I started using primary/secondary colours as orange/blue, this was ruined. So this adds the `darkGrey` to the `color` prop of the checkbox 🤷

Reply:
yeah, I have hovered on doing the same thing. wasn't sure that secondary should be blue throughout, I guess we'll find out as we go on which will be the more useful. I thought that having the grey as a secondary would fit with the mui components.. 🤷‍♂️